djangocms_blog/cms_plugins.py

```diff
@@ -17,15 +18,18 @@ class BlogPlugin(CMSPluginBase):
     module = get_setting('PLUGIN_MODULE_NAME')
 
     def get_render_template(self, context, instance, placeholder):
+
+        templates = [os.path.join('djangocms_blog',
+                                  instance.template_folder,
+                                  self.base_render_template)]
+
         if instance.app_config and instance.app_config.template_prefix:
-            return os.path.join(instance.app_config.template_prefix,
-                                instance.template_folder,
-                                self.base_render_template)
-        else:
-            return os.path.join('djangocms_blog',
-                                instance.template_folder,
-                                self.base_render_template)
+            templates.insert(0, os.path.join(instance.app_config.template_prefix,
+                                             instance.template_folder,
+                                             self.base_render_template))
 
+        selected = select_template(templates)
```

Comment:
This trigger a failure in a test (https://travis-ci.org/nephila/djangocms-blog/jobs/386689256#L477)
I think the current test does not expect any check on the actual template name, thus probably some fake templates must be added
I'd restructure the test to also test this change

Reply:
Sorry for the long delay.  Spam filter was eating my GitHub notices.  I'll put this back on my to-do list.  Just hit this issue again in another place with the tags plugin.